csharp/README.dist

```diff
@@ -3,18 +3,17 @@ XenServer.NET
 
 Version @SDK_VERSION@
 
-XenServer.NET is a complete SDK for Citrix XenServer,
-exposing the XenServer API as .NET classes. It is written in C#.
+XenServer.NET is a complete SDK for the Citrix Hypervisor,
+exposing the Citrix Hypervisor API as .NET classes. It is written in C#.
 
-It is available in the XenServer-SDK-@SDK_VERSION@.zip, which can be downloaded
-from http://www.citrix.com/downloads/xenserver/.
+It is available in the CitrixHypervisor-SDK-@SDK_VERSION@.zip, which can be downloaded
+from http://www.citrix.com/downloads/citrix-hypervisor/.

```

Comment:
Change to "https", although "http" also works by redirecting. Better to align with the same URL above.